include/envoy/ssl/certificate_validation_context_config.h

```diff
@@ -54,6 +54,11 @@ class CertificateValidationContextConfig {
    * @return whether to ignore expired certificates (both too new and too old).
    */
   virtual bool allowExpiredCertificate() const PURE;
+
+  /**
+   * @return True if client certificate does not require validation, false otherwise.
```

Comment:
s/True/true

I also think that "does not require validation" sounds like an implementation detail so it might be better to rephrase that.